entity-framework/core/extensions/index.md

```diff
@@ -363,7 +363,7 @@ It also has many useful features commonly used especially on web development. Fo
 ### Laraue.EfCoreTriggers
 Fluent API to declare triggers in `Context.OnModelCreating` which are later built into migrations. Providers to Postgres, MySql, SqlServer and SqlLite.
 
-There is some support for method call expression conversion. The library is on par with framework releases and the devoloper seems engaged.
+Support for EFCore:  5, 6.
```

Comment:
Please use the same format as everywhere else in this file - so right after the description, add:  `For EF Core: 5, 6.`